clean-base.py

```diff
@@ -1,63 +1,72 @@
 #!/usr/bin/env python3
-import argparse, datetime, getpass, io, os, subprocess, sys, time, zipfile
+import argparse
+import datetime
+import getpass
+import io
+import os
+import subprocess
+import sys
+import time
+import zipfile
+
 from urllib.request import urlopen
 
 VI_VERSION = "3.0.20"
 
 try:
-	whoami = getpass.getuser()
+    whoami = getpass.getuser()
 except:
```

Comment:
Related to [E722](https://www.flake8rules.com/rules/E722.html), we shouldn't use bare excepts. 

tbh, it's not really clear which are the possible exception ([_an exception is raised_](https://docs.python.org/3/library/getpass.html#:~:text=an%20exception%20is%20raised)) Therefor use `Exception`.

```suggestion
except Exception:
```